build.sbt

```diff
@@ -182,6 +185,28 @@ lazy val refined = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .nativeSettings(sharedNativeSettings)
   .dependsOn(core)
 
+lazy val iron = crossProject(JVMPlatform)
+  .in(file("modules/iron"))
+  .settings(
+    moduleName := "ciris-iron",
+    name := moduleName.value,
+    dependencySettings ++ Seq(
+      libraryDependencies := Seq(
+        "io.github.iltotore" %% "iron" % ironVersion,
```

Comment:
In order to support JS and Native, you should use `%%%` instead of `%%` for Iron dependency.